docs/INPUT_FILES_EXPLAINED.md

```diff
@@ -0,0 +1,92 @@
+# Model Input Files
+
+- [Primary Configuration File](#primary-configuration-file)
+- [Input Data](#input-data)
+- [Subcatchment Data](#subcatchment-data)
+- [Parameter Data](#parameter-data)
+
+The header file [`topmodel.h`](../include/topmodel.h) also includes comments of each variable, both input and output.
+
+## Primary Configuration File
+See [`topmod.run`](../data/topmod.run) for Topmodel's main configuration file.
+The image and table below explain the file's contents
+All entry values can be customized, but must remain the same order for proper read-in during `bmi.initialize()`.
+Note that the first `Boolean` `stand_alone` flag is specific to BMI extention and is not present in source code.
+More on `stand_alone` can be found [here](./STAND_ALONE.md).
+
+<p align="center">
+  <img src="https://github.com/madMatchstick/topmodel/blob/doc-update-bmi-v2.0/docs/img/topmod_run.PNG" width=45% height=45% >
+</p>
+
+| Variable | Datatype | Limits | Role | Description |
+| -------- | -------- | ------ | ---- | ----------- |
+| stand_alone | *int* | Boolean | option | `1`(`TRUE`) or `0` (`FALSE`) flag to indicate if model is to be run via *stand alone* mode; added for BMI adaption |
+| title          | *char* | 256 | info_string | character title of model |
+| \*input_fptr   | FILE   | 256 | file_offset | file pointer to `inputs.DAT`  |
+| \*subcat_fptr  | FILE   | 256 | file_offset | file pointer to `subcat.DAT`  |
+| \*params_fptr  | FILE   | 256 | file_offset | file pointer to `params.DAT`  |
+| \*output_fptr  | FILE   | 256 | file_offset | file pointer to `topmod.OUT`  |
+| \*out_hyd_fptr | FILE   | 256 | file_offset | file pointer to `hyd.OUT`     |
+
+Note\*: All file pointers must be a full relative path; e.g. `data/inputs.DAT`.
+
+## Input Data
+See [`inputs.dat`](../data/inputs.dat).
+
+<p align="center">
+  <img src="https://github.com/madMatchstick/topmodel/blob/doc-update-bmi-v2.0/docs/img/inputs_dat.PNG" width=45% height=45% >
+</p>
+
+| Variable | Datatype | Units | Role | Description |
+| -------- | -------- | ----- | ---- | ----------- |
+| nstep  | *int* |   | time_info | total number of simulation periods |
+| dt     | *int* | hours  | time_info | size of timestep  |
+| rain   | *double* |  meters/hour | input_from_file\* | rainfall rate  |
+| pe     | *double* |  meters/hour | input_from_file\* | potential evapotranspiration  |
+| Qobs   | *double* |  meters/hour | input_from_file\* | observed discharge   |
+
+Note\*: Variable role is "input_from_bmi" if not in stand-alone mode.
+
+## Subcatchment Data
```

Comment:
Is this the file that can be generated using the params workflow?  If so, this would be a good place to specify.